src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-dcgm-integration.mdx

```diff
@@ -0,0 +1,125 @@
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/).
+
+## Configure DCGM-Exporter
+1. Clone the repository
+```
+git clone https://github.com/NVIDIA/dcgm-exporter
+```
+2. Below command is used to open the `dcgm-exporter` directory present in the cloned repository.
+```
+cd dcgm-exporter
+```
+3. Install necessary binaries using below commands.
+```
+make binary
+```
+
+```
+sudo make install
+```
+4. Start the dcgm-exporter using below command
+```
+dcgm-exporter &
+```
+5. Use the below command to see details of dcgm metrics 
+```
+curl localhost:9400/metrics
+```
+
+## NVIDIA-DCGM configuration on Prometheus
+
+To install Prometheus on a Linux system, you can follow these steps. Prometheus is a popular open-source monitoring and alerting tool.
+
+1. Visit the [Prometheus download page](https://prometheus.io/download/) to find the latest release.
+2. Select the appropriate version for your operating system and architecture. For Linux, you'll likely choose the linux-amd64 version. Copy the download link for the tarball (.tar.gz file).
+3. Once downloaded Prometheus, untar the download tar file.
+```shell
+tar -xvzf <filename.tar.gz>
+```
+4. Go to the Prometheus folder
+```shell
+cd <download folder>
+```
+5. Open your `Prometheus.yml` file and add the following lines.
+```yml
+  ---
+  scrape_configs:
+    - job_name: NVIDI
+      static_configs:
+        - targets:['localhost:9400']
+```
+6. Start the Prometheus using the below command
+```shell
+./prometheus --config.file=prometheus.yml
+```
+
+## Configure Prometheus remote write for NVIDIA DCGM
+
+1. Set up Prometheus monitoring. Prometheus metrics needs to be integrated with New Relic, you can use the Remote Write integration by following [Prometheus remote write setup launcher in the UI](https://one.newrelic.com/catalog-pack-details?state=04de330d-ad8b-b61c-68e2-ad852c2ca1d4).
+
+2. Using the Prometheus Remote Write integration, you can point your Prometheus servers at New Relic to store and visualize your data. You can explore other ways to integrate Prometheus data [in our docs](https://docs.newrelic.com/docs/integrations/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/).
```

Comment:
@RamanaReddy8801 Is this required or optional? Is it a continuation of the previous Prometheus setup instructions? Let's add some information here about why you'd want/need to configure Prometheus remote write and how it's connected to the previous section.

Reply:
Yes, It is continuation of the above step and  I added some information it will help user to understand.